src/main/java/com/qaprosoft/carina/demo/guipractice/pages/HomePage.java

```diff
@@ -12,32 +12,31 @@
 import com.qaprosoft.carina.demo.guipractice.components.Product;
 
 public class HomePage extends AbstractPage {
-	
+
 	@FindBy(xpath = "//nav[@id = 'narvbarx']")
 	private NavigateMenu navigateMenu;
-	
+
 	@FindBy(xpath = "//*[@id = 'tbodyid']")
 	private List<Product> products;
```

Comment:
wrong locator here. For components we should specify locator only for one instance of component on the page, not for the block with all objects
Here you need to specify locator for this block
<img width="277" alt="image" src="https://user-images.githubusercontent.com/103670717/164194618-86f52c5b-334f-456a-be2b-f45e27a0cfbc.png">